src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala

```diff
@@ -0,0 +1,173 @@
+package is.hail.annotations.aggregators
+
+import is.hail.annotations._
+
+class RegionValueMinBooleanAggregator extends RegionValueAggregator {
+  private var min: Boolean = false
+  private var empty: Boolean = true
+
+  def seqOp(i: Boolean, missing: Boolean) {
+    if (!missing && (i < min || empty)) {
```

Comment:
would prefer `empty || i < min` for all of these, so the `i < min` check isn't redundant for the `empty` case.

Reply:
I thought I was saving operations by doing it this way bc the common case is `empty` will be true.

Reply:
@cseed any advice on `||` versus `|` on simple Boolean expressions like above?

Reply:
ok. that's probably true.

Reply:
Meant `empty` will be false in most cases.